serverless.yml

```diff
@@ -0,0 +1,29 @@
+service: ${file(./serverless-base.yml):service}
+frameworkVersion: ${file(./serverless-base.yml):frameworkVersion}
+package: ${file(./serverless-base.yml):package}
+functions: ${file(./serverless-base.yml):functions}
+
+provider:
+  name: ${file(./serverless-base.yml):provider.name}
+  runtime: ${file(./serverless-base.yml):provider.runtime}
+  memorySize: 512
+  timeout: 30
```

Comment:
A thought for later, might want to increase this, depending on how long we think it might take for all the processing to happen. I think the max is 15 mins.